packages/block-library/src/comment-template/edit.js

```diff
@@ -119,12 +124,52 @@ export default function CommentTemplateEdit( {
 	const blockProps = useBlockProps();
 
 	const [ activeComment, setActiveComment ] = useState();
-	const { commentOrder, commentsPerPage } = useSelect( ( select ) => {
+	const {
+		commentOrder,
+		commentsPerPage,
+		threadCommentsDepth,
+		threadComments,
+	} = useSelect( ( select ) => {
 		const { getSettings } = select( blockEditorStore );
 		return getSettings().__experimentalDiscussionSettings;
 	} );
 	const { rawComments, blocks } = useSelect(
 		( select ) => {
+			// Show placeholder comments if we don't have a postId context.
+			// The structure of rawComment as an empty object allows
+			// inner blocks to render the default placeholders.
+			if ( ! postId ) {
+				// If displaying threaded comments is disabled, we only show one comment
+				if ( ! threadComments ) {
+					return { rawComments: [ { id: null } ] };
```

Comment:
Should the `id` be different here `0` or `-1`? @michalczaplinski should know better.

Reply:
It should probably not be `0` because that can be a valid `id`, right? However, I think it does not matter probably, because we should find a way not to to make the calls to the REST API to fetch the "fake" comments at all like you and Carlos have [found out already](https://github.com/WordPress/gutenberg/pull/38072#discussion_r795535875) 😄